OnlineLibrary.Common/ErrorMiddleware/ErrorHandlerMiddleware.cs

```diff
@@ -0,0 +1,46 @@
+﻿using Microsoft.AspNetCore.Http;
+using OnlineLibrary.Common.Exceptions;
```

Comment:
think about what to use when middleware or filters 
In that case, it's also possible to use a global exception filter